index.js

```diff
@@ -31,11 +32,10 @@ function NYC (opts) {
   config = config.nyc || {}
 
   // load exclude stanza from config.
-  this.exclude = config.exclude || ['node_modules[\/\\\\]', 'test[\/\\\\]', 'test\\.js']
+  this.include = this._prepGlobPatterns(config.include)
+  this.exclude = ['**/node_modules/**'].concat(config.exclude || ['test/**', 'test{,-*}.js'])
```

Comment:
The next line is no longer necessary, right? Which means you could wrap it with `this._prepGlobPatterns()` and only assign `this.exclude` once.